misc/vtk/include/inviwo/vtk/processors/vtkdatasetinformation.h

```diff
@@ -84,10 +84,9 @@ class IVW_MODULE_VTK_API VTKDataSetInformation : public Processor {
     public:
         ArrayInformationProperty() = delete;
         ArrayInformationProperty(const std::string& arrayName, const std::string& identifier,
-                                 const vtkDataArray* array)
+                                 vtkDataArray* array)
             : CompositeProperty(identifier, arrayName)
-            , dataType_(identifier + "dataType", "Data type",
-                        std::string{array->GetDataTypeAsString()})
+            , dataType_(identifier + "dataType", "Data type", array->GetDataTypeAsString())
```

Comment:
Why is this inline? why not in the c++? 

Might remove the doc at the top too...